src/gapoints.test.ts

```diff
@@ -0,0 +1,37 @@
+import * as GA from "./gapoints";
+import * as GALine from "./galines";
+
+describe("Test gapoints", () => {
+  describe('toTuple', () => {
+    it("should extract coords from a NVector representing the origin", () => {
+      expect(GA.toTuple([0, 0, 0, 0, 0, 0, 1, 0])).toEqual([0,0]);
+    });
+
+    it("should extract coords from a NVector representing a nonorigin point", () => {
+      expect(GA.toTuple([0, 0, 0, 0, 1, 1, 1, 0])).toEqual([1,1]);
+    });
+
+    it("should extract coords from an unnormalized NVector representing a nonorigin point", () => {
+      expect(GA.toTuple([0, 0, 0, 0, 2, 2, 2, 0])).toEqual([1,1]);
+    });
+
+    it("should extract coords from a unnormalized negated NVector representing a nonorigin point", () => {
+      expect(GA.toTuple([0, 0, 0, 0, -6, -8, -2, 0])).toEqual([4,3]);
+    });
+    it("should extract coords from a normalized negated NVector representing a nonorigin point", () => {
+      expect(GA.toTuple([0, 0, 0, 0, 1, 0, -1, 0])).toEqual([-0,-1]);
+    });
+  });
+
+  describe('intersect', () => {
+    it('should intersect pairs of points', () => {
+      expect(GA.toTuple(GA.intersect(
+        GALine.through(GA.from([1, 1]), GA.from([1, 2])), 
+        GALine.through(GA.from([1, 1]), GA.from([2, 1])), 
+      ))).toEqual([1, 1]);
```

Comment:
This was returning (-1, 1) previously.

Reply:
Hard to believe, i know, but you can run it for yourself:

<img width="762" alt="Screen Shot 2023-05-01 at 3 32 50 PM" src="https://user-images.githubusercontent.com/15707615/235543078-b4b6038e-106f-4709-bdb9-5cb93c323d4f.png">